app/deed/server.py

```diff
@@ -72,6 +68,22 @@ def create():
             }
         }
 
+        borrowers = case_api.get_borrowers(deed_json['case_id']).json()
+
+        for borrower in borrowers:
+            borrower["token"] = Deed.generate_token()
+            json_doc["deed"]["operative-deed"]["borrowers"].append(borrower)
+
+        property = case_api.get_property(deed_json['case_id']).json()
+        title_json = {"title-number": property['title_number'],
+                      "address": {"street-address": property['street'],
+                                  "postal-code": property['postcode'],
+                                  "locality": property['locality'],
+                                  "extended-address": property.get('extended')
+                                  }}
+
```

Comment:
Maybe we could modify get_property endpoint in case_api to return json matching the one in deed? This way we could avoid this transformation.